apps/src/lib/levelbuilder/lesson-editor/ActivitiesEditor.jsx

```diff
@@ -123,12 +123,15 @@ class ActivitiesEditor extends Component {
             />
           ))}
           <button
-            onMouseDown={this.handleAddActivity}
+            onClick={this.handleAddActivity}
```

Comment:
A bunch of stuff is onMouseDown instead of onClick because onClick was causing the page to save. Can you check manually that this does not cause that issue?

Reply:
that seems like a problem which will be fixed when we move to doing updates via ajax instead of form post. also your other PR is going to make it so that there is always an activity, after which I won't need to click this button. so once that PR is in I'll change this back to onMouseDown and then send for re-review.